renovate-base.json5

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
perhaps adding a comment explaining how this config is used? for the future us and maintainers :)

Reply:
actually something really similar to the PR perhaps

Reply:
I'm including some notes in the README, thanks for pointing it out!

Reply:
Updated here: [c1d322f](https://github.com/newrelic/coreint-automation/pull/13/commits/c1d322f88c1185305dd358d6e5ffd19baee9d369)